playbooks/utils/clean_up_alma_dir.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Delete oldest files from /alma directories
+  hosts: libsftp_{{ runtime_env | default 'production'}}
```

Comment:
This defaulting to production when all our other playbooks default to staging makes me nervous.

Reply:
Heh, yeah, I caught that the first time I tested the playbook. It's fixed now, thanks for the vigilance!